src/main/java/pages/MainPage.java

```diff
@@ -57,8 +57,14 @@ public void selectFromDropDown(String element) {
 
     @Override
     public void hoverOverTheElement(String element) {
+        try{
+            Thread.sleep(1000);
+            moveToElementAndClick(getWebElement(map, element));
+        }catch (Exception ex){
+
+        }
+
```

Comment:
Thread sleep might not be the best solution. Try to use implicitly wait function